src/test/java/uk/gov/hmcts/reform/sandl/snlrules/JwtTokenHelper.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.sandl.snlrules;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.http.HttpHeaders;
+import uk.gov.hmcts.reform.sandl.snlrules.security.S2SAuthenticationService;
+
+import java.util.Date;
+
+public class JwtTokenHelper {
+
```

Comment:
This class should probably be removed as I don't see any usages.

Reply:
it is by integration tests, moved there.